Exception/KaitaiStructError.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Kaitai
+{
+    /// <summary>
+    ///     Common ancestor for all error originating from Kaitai Struct usage.
+    ///     Stores KSY source path, pointing to an element supposedly guilty of
+    ///     an error.
+    /// </summary>
```

Comment:
I've noticed that you indent XML comments with one extra 4-space indent in `<summary>...</summary>` — can you point me to some kind of official standard regarding that?

Reply:
Nice catch! I use Resharper and it does that by default. https://stackoverflow.com/questions/16533233/resharper-indenting-the-xml-documentation-text-on-code-cleanup

Reverted.